elasticapm/instrumentation/packages/base.py

```diff
@@ -208,6 +208,16 @@ def instrument(self):
                          self.name, ex)
         self.instrumented = True
 
+    def uninstrument(self):
+        if not self.instrumented or not self.originals:
+            return
+        for module, method in self.get_instrument_list():
+            if (module, method) in self.originals:
```

Comment:
How about looping over `self.originals` instead?

Reply:
my goal was to mirror `instrument()` as closely as possible. Looping over `self.originals` is slightly more efficient in some cases (when we only were able to wrap part of the instrument list), but I think mirroring the logic of `instrument()` has more value when it comes to readability.